examples/bri-1/lib/proubc-abap-connector/PROUBC-intro.md

```diff
@@ -0,0 +1,21 @@
+# proUBC
+*An open source, SAP ABAP connector for PRVD Baseline aka BRI-1*
```

Comment:
Please do not introduce pseudonymous nomenclature for bri-1.

Reply:
We will review naming things.... this is definitely misleading since BRI-1 is actually meant to reference an end-to-end test harness.

proUBC functionality can absolutely be part of the BRI-1 harness but BRI-1 I would not consider "PRVD Baseline" -- it's really just **_Baseline by way of the PRVD stack_**, which is maintained under PRVD OASIS.